src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double firstSide;
+    private double secondSide;
+
+    public Rectangle(Color color, double firstSide, double secondSide) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public void print() {
+        System.out.println("Figure: " + FigureName.RECTANGLE.name()
+                + ", area:" + getArea()
+                + ", sq.units, first side:" + firstSide
+                + ", units, second side: " + secondSide
+                + ", units, color: " + getColor());
```

Comment:
[To get a string representation of an enum use `name()`](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)